### DISCUSSION TITLE
Shared workflow is unable to be called from a personal/user repository

### DISCUSSION THREAD

**Discussion by jmcconathy** [community/community#121692] [Enterprise]
In a GitHub Enterprise server (3.12.2), we have shared workflows stored in one organization.  We have been using these shared workflows from other organizations in our enterprise without issue.  Yesterday I tried using one of the shared workflows from within a personal repository, that is, a repository within my user profile on the GitHub Enterprise Server.

When I made an action that called the shared workflow, I got the following error: 
**Invalid workflow file.  The workflow is not valid ... Error calling workflow 'myworkflow'.  The nested job 'myjob' is requesting 'id-token: write', but is only allowed 'id-token: none'.** 

I tried a number of things with the settings on my repository but none eliminated this error, so I explicitly set permissions for the workflow token to be all write.  

That eliminated that error, but then I get this one:
**Called workflows cannot be queued onto self-hosted runners across organizations/enterprises.  Failed to queue this job.**

I have tried changing my repo settings to make it as permissive as possible, but nothing gets past this error.

What else could be blocking my personal repo from accessing a shared workflow?

The workflow repos are set to Public, and work just fine when called from repos in other Orgs of our enterprise.  And I have runners setup for my repo that use the same labels being needed by the shared workflows I want to call.

**Comment by robertIsaac** [comment]
+1
